src/dragon.rs

```diff
@@ -0,0 +1,37 @@
+/// Original dragon image modified from:
+/// https://www.rawpixel.com/image/581164/hand-drawn-dragon
+/// using the following webiste:
+/// https://www.text-image.com/convert/ascii.html
+pub fn loading(){
```

Comment:
nit: space before brace
```suggestion
pub fn loading() {
```